x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/blockhash/BlockHashTests.java

```diff
@@ -1232,6 +1235,190 @@ public void testLongNull() {
         }, blockFactory.newLongArrayVector(values, values.length).asBlock(), blockFactory.newConstantNullBlock(values.length));
     }
 
+    public void test2BytesRefsKeys() {
```

Comment:
nit: maybe a more descriptive name? Like `test2BytesRefsHighCardinalityKey` or so?

Reply:
:+1: on `HighCardinality`.

Reply:
This is java. Lean into the memes about long method names....